prog/vm/host_nexus.rb

```diff
@@ -48,7 +48,9 @@ def self.assemble(sshable_hostname, location_id: Location::HETZNER_FSN1_ID, fami
 
   label def setup_ssh_keys
     # Generate a new SSH key if one is not set.
-    sshable.update(raw_private_key_1: SshKey.generate.keypair) unless sshable.raw_private_key_1
+    unless sshable.raw_private_key_1
+      sshable.update(raw_private_key_1: SshKey.generate.keypair)
+    end
```

Comment:
No objection, but this change seems unrelated to the rest of the commit.

Reply:
ai dross from the churn, I'll purge it